tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "target": "es5",
+        "module": "commonjs",
+        "skipLibCheck": true,
+        "strict": true,
+        "types": [
+            "cypress"
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts"
+    ],
+    "exclude": [
+        "node_modules/"
+    ]
+}
```

Comment:
EOF

Reply:
Hi Holmes, I updated the file with tab (EOF)